src/helics/CMakeLists.txt

```diff
@@ -50,6 +50,11 @@ if(BUILD_SHARED_LIBS)
       $<INSTALL_INTERFACE:$<INSTALL_PREFIX>/include>
   )
 
+  set_target_properties(helics-shared PROPERTIES
+    VERSION ${HELICS_VERSION}
+    SOVERSION ${HELICS_VERSION_MAJOR}
```

Comment:
does the SOVERSION not need to change for minor version updates?  Or is this actually about backwards compatibility

Reply:
It’s backwards compatibility. The SONAME is set in all of the shared libraries to sharedLib.so.<SOVERSION> and is what a program actually links to, so a new major version could be installed side-by-side the old one without breaking programs that are linked to the older version.